apps/angular/src/core/models/anime.ts

```diff
@@ -0,0 +1,47 @@
+import { AnimeStatus } from './anime-status';
+import { AnimeType } from './anime-type';
+
+import { DateTimeRange } from './date-time-range';
+
+/** Represents anime. */
+export type Anime = {
+
+	/** Id. */
+	readonly id: number;
+
+	/** Created. */
+	readonly created: Date;
+
+	/** Modified. */
+	readonly modified: Date;
+
+	/** Anime title in English. */
+	readonly titleEng: string;
+
+	/** Anime title in Japanese. */
+	readonly titleJpn: string;
```

Comment:
Please, [avoid](https://wiki.saritasa.rocks/frontend/languages/ts-js/naming#avoid-contractions) contractions:
```suggestion
	/** Anime title in English. */
	readonly titleEnglish: string;

	/** Anime title in Japanese. */
	readonly titleJapanese: string;
```